### PR TITLE
Add Game Tracker, RedBooth, RedFin and SyncFusion as easy

### PR DESCRIPTION
**Pull request by alrjunior**
Add Game Tracker, RedBooth, RedFin and SyncFusion (all of them easy).

### CODE REVIEW

File: _data/sites.json [hunk 1]
```diff
@@ -5533,6 +5533,18 @@
         ]
     },
 
+    {
+        "name": "Game Tracker",
+        "url": "http://www.gametracker.com/account/manage/edit.php",
+        "difficulty": "easy",
+        "notes": "Scroll down in the account manager and click on close account button. 30 days after closing the account, it will be deleted.",
```
Comment (tupaschoal) [inline _data/sites.json:5540]:
```suggestion
        "notes": "Scroll down in the account manager and click on close account button. The account will be deleted 30 days after closing it.",
```

File: _data/sites.json [hunk 2]
```diff
@@ -5533,6 +5533,18 @@
...
+        "notes_fr": "Descendez dans les paramètres, cliquez sur le bouton close account, votre compte sera alors définitivement supprimé dans un délai de 30 jours.",
+        "notes_pt_br": "Desça na página da conta e clique em close account. Após 30 dias, a conta será definitivamente deletada.",
```
Comment (tupaschoal) [inline _data/sites.json:5542]:
```suggestion
        "notes_pt_br": "Desça na página da conta e clique em 'close account'. A conta será deletada definitivamente após 30 dias.",
```

File: _data/sites.json [hunk 3]
```diff
@@ -12478,6 +12500,16 @@
         ]
     },
 
+    {
+        "name": "Redfin",
+        "url": "https://www.redfin.com/myredfin/settings",
+        "difficulty": "easy",
+        "notes": "After logging in, click the Deactivate button at the bottom of the Settings page.",
```
Comment (tupaschoal) [inline _data/sites.json:12507]: Do you know if this leads to deletion or just deactivation? i.e. can you reactivate it?

File: _data/sites.json [hunk 4]
```diff
@@ -14241,6 +14273,16 @@
         ]
     },
 
+    {
+        "name": "Syncfusion",
+        "url": "https://www.syncfusion.com/account/my-profile",
+        "difficulty": "easy",
+        "notes": "In \"My Profile Page\", click in \"Delete Account\" link at left and click in \"Delete Account\" button.",
```
Comment (tupaschoal) [inline _data/sites.json:14280]:
```suggestion
        "notes": "In \"My Profile Page\", click on \"Delete Account\" link on the left side and then the \"Delete Account\" button.",
```

File: _data/sites.json (hunk 3 above)
Comment (alrjunior) [reply _data/sites.json]: It leads to the deletion of the account. I've just fixed the note.

**Review by tupaschoal — APPROVED** [review]

**Comment by tupaschoal** [comment]
Thanks for your contribution!

**Pull request merged**